subprojects/core/src/main/java/org/gradle/process/internal/streams/StreamsForwarder.java

```diff
@@ -62,11 +63,15 @@ public void connectStreams(Process process, String processName, ExecutorFactory
     }
 
     public void start() {
-        executor.execute(standardInputRunner);
+        executor.execute(wrapInBuildOperation(standardInputRunner));
```

Comment:
does it make sense to wrap standard input because it's not going to be logged at all?